rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -71,20 +77,22 @@ class DecodingRules():
     address_mappings: dict[ChecksumEvmAddress, tuple[Any, ...]]
     event_rules: list[EventDecoderFunction]
     token_enricher_rules: list[Callable]  # enrichers to run for token transfers
-    post_decoding_rules: list[tuple[int, Callable]]  # rules to run after the main decoding loop
+    post_decoding_rules: dict[str, list[tuple[int, Callable]]]  # noqa: E501 rules to run after the main decoding loop
```

Comment:
Perhaps explain here that it's a mapping of counterparty to tuple post decoder function